### ISSUE TITLE
Segmentation 11 - YES AGAIN!

### ISSUE THREAD

**Issue by zois** [IvanVorobei/RequestPermission#30]
Although it's a nice pod, it needs tons of work regarding its pod handling.
When updating pod, I get that I am at version 1.0.0 whereas previous version was 1.0.1!!! When I remove the pod and reinstall it I get segmentation 11 fault when building. I checked the answers to the other closed issues but really no professional solution has been proposed. A guy deletes a swift file from the pod and uses it's code directly. Not a good way. The author just denies to fix this. And no, I won't zip and send you anything from my machine for you to check.
If you fix this, I 'll try this pod again, otherwise I 'll have to use another permission pod from out there...

Regards

**Comment by IvanVorobei** [comment]
If you can not zip the project and send it to me - how can I help you? I do not know how to guess thats happen 

I can not reproduce the problem and help you. I'm truly sorry. I will be happy if you use another pod and will be happy with it!

Or you can be more calm and share with me all the information so that I can understand what the problem is. Once again I repeat, under the integrated in hundreds of devices ... no one has problems

Regards, @zois

**Comment by zois** [comment]
It can certainly be reproduced cause this has happened to many users and we all say the same thing. Specifically, when we update the pod and then we build we get this segmentation fault : 11.
That's all.
Also the pod update process always says version as 1.0.0.
Initially when I first installed it I didn't have problems either! After an update I got this issue.

This is only solved by uninstalling the pod and commenting the referenced code so the project can build. Then install the pod again and uncomment the code that uses it.

This is the second time I had to do this procedure for your pod. Pods were meant to be an easy helper for a developer, not a hard time. You guys make great features and you don't pay attention at all to properly setting them up as pods.

Regards
